Packages/doc/releasenotes_new.rst

```diff
@@ -0,0 +1,244 @@
+44253e11 (Merge pull request #924 from AllenInstitute/feature/924-add-branch-naming-scheme-documentation, 2021-04-23)
+
+Release 2.2
+===========
+
+This version is the first version of MIES with support for upcoming Igor Pro 9. And it will also be the last with
+support for Igor Pro 8.
+
+Sweep Formula
+-------------
+
+- Add ``area`` operation
+- Fix most cases of ignored minus signs in formulas
+- Make it more user friendly on errors by not just asserting out
+
+
+AnalysisBrowser
+---------------
+
+
+DataBrowser
+-----------
+
+Dashboard
+~~~~~~~~~
+
+- Display the headstages for each sweep
+- Add support for multipatch seq analysis functions
+
+
+DataBrowser/SweepBrowser
+------------------------
+
+- Fix restoring from backup so that it works again and also restores all sweeps and not only the displayed ones
+- The axes' locations are now more predictable for zomming in/out via mouse wheel
+- Don't display data from TP during DAQ channels
+- Always create backup waves when splitting the sweep
+- Speedup plotting of many traces by grouping AppendToGraph calls. This is mostly noticable in Igor Pro 8, les in
+  Igor Pro 9.
+- Make overlay sweeps with headstage removal faster
+- Select the current sweep when enabling overlay sweeps
+- Add new trace popup for convenient stimulus set opening in the wavebuilder
+- Fix displaying TTL data with only "TP during DAQ" data
+
+
+DA\_Ephys
```

Comment:
DA_Ephys

Reply:
The `\` is required for sphinx, it is not visiable in the output, see https://alleninstitute.github.io/MIES/releasenotes.html#da-ephys.